frontend/src/components/messstelle/charts/MessstelleDiagramme.vue

```diff
@@ -0,0 +1,395 @@
+<template>
```

Comment:
bevor ich das komplett review, würde ich hier vorschlagen nen clean code Ansatz zu verwenden und das ganze sehr viel kleinteiliger aufzubrechen. Bei den Zählstellen hat das ding über 1000 Zeilen, das würde ich hier gern vermeiden. 

Daher eine eigene komponente für die Kopfzeile / jeden Tab-inhalt / speed-dial und funktionaltiät die 1:1 rüberkopiert wurde in composables stecken und importieren

Reply:
Speeddial habe ich rausgezogen. Alles andere muss so bleiben wie es ist, da sonst die Funktionalität nicht mehr gewährleistet ist. Ich ziehe aber wenn möglich noch ein paar der Reportmethoden in ein composable raus.

Reply:
bitte noch pushen